source/patterns/02-organisms/content-header/content-header~ar.yaml

```diff
@@ -3,3 +3,8 @@ attributes:
   lang: ar
 contentTitle:
   text: عنوان المحتوى
+downloadIconLink:
+  attributes:
```

Comment:
I'd rename `downloadIconLink` to `download` or `downloadLink`, then don't have a separate `attributes`, so just:

```yaml
download:
    href: '#'
```

This will have to change when we introduce the menu.

Reply:
`attributes` layer needed for https://github.com/libero/pattern-library/pull/98#discussion_r293740807?

Reply:
No, that can still exist (`download` being the list of attributes).

Reply:
See 752e6db8f0c7001f010f878fa1f3de939d14ce4c.